### ISSUE TITLE
[feature request] VueJS Roadmap and RFC

### ISSUE THREAD

**Issue by 17twenty** [ricokahler/hacker-ui#8]
**Is your feature request related to a problem? Please describe.**

I would love to see this become a default for modern web app development and think a VueJS version would be great.

**Describe the solution you'd like**

Layout ground work and recruit interested community members to work on the Vue version.

**Describe alternatives you've considered**
Vuetify / Buefy are the two best known toolkits at the moment (leveraging both Material and Bulma).

**Comment by ricokahler** [comment]
I would also love to see this become a default for web apps! You are also not the first person to ask if there were any plans to support Vue and you definitely won't be the last.

I would be very open to spinning up efforts to support Vue, however, in an effort to ship incrementally, I could only see myself starting that effort after this 1.0 (see #6) in about 2-3 months when Hacker UI is in a more stable state and has gain a bit of popularity.

This is coming from personal experience in saying yes to too many features/increasing scope too often and thus never releasing anything.

I know you're not exactly asking me to implement everything in Vue but there's still a lot of coordination and configuration required to get the effort going. E.g. How do we port the CSS-in-JS system? do we create a scope in NPM?, do we put this code in a mono repo or under and organization? how do we sync up changes between both versions? etc

---

In short, I would love to support Vue ❤️ but Hacker UI isn't defined enough to start that effort.

Thanks for opening this issue! It brings visibility to demand for Vue support. I'll leave this open in case anyone else has this same question.
